controllers/version.go

```diff
@@ -19,10 +19,10 @@ package controllers
 import "regexp"
 
 const (
-	supportedIstioMinorVersionRegex = "^1.11"
+	supportedIstioMinorVersionRegex = "^1\\.11"
```

Comment:
What do you think about this? Just to account for sth like `1.11.4-rc.0`, but not allow sth like `1.11whatever`.

```suggestion
	supportedIstioMinorVersionRegex = "^1\\.11(\\.[0-9]+(?:-(?:dev|rc)\\.[0-9]+)?)?$"
```

Reply:
What about `^1\\.11(\\.[0-9]+)?(-.+)?$` that should cover any any tag involving 1.11 (1.11.2, 1.11.2-test, 1.11-test, etc.) ?

Reply:
Sounds good as well, I guess it also matches `1.11.1-dev.` though.

Reply:
Should cover every possible 1.11 semver tag. Modifying PR....